modules/nf-core/smoove/call/main.nf

```diff
@@ -22,13 +22,16 @@ process SMOOVE {
     script:
     def args = task.ext.args ?: ''
     def prefix = task.ext.prefix ?: "${meta.id}"
+
+    def exclude = exclude_beds ? "--exclude ${exclude_beds}" : ""
```

Comment:
Shouldn't the exclude_beds be optional?

Reply:
This line makes them optional. If you use `[]` as input for `exclude_beds` it won't use any :)

Reply:
Ok, perfect.
Thanks